app/models/course_user.rb

```diff
@@ -119,6 +119,17 @@ def on_rejected_entry(*)
     destroy
   end
 
+  # Returns my students in the course.
+  # If a course_user is the manager of a group, all other users in the group with the group role of
+  # normal will be considered as the students of the course_user.
+  #
+  # @return[Array<CourseUser>]
+  def my_students
```

Comment:
It's either doing it when we are adding `group_users`, or having to know which scopes to apply to when any resulting component requires `group.group_users`. 

Both are fine, but as we do more work involving groups, it seems like we have to make a few assumptions about groups - non-approved course_users cannot be enrolled, etc.